docs/content/en/docs/clustermgmt/cluster-scale/cloudstack-scale.md

```diff
@@ -31,22 +31,9 @@ spec:
   - count: 1     # increase this number to horizontally scale your data plane
 ```
 
-Once you have made configuration updates you can apply the changes to your cluster.
+Once you have made configuration updates, you can use `eksctl`, `kubectl`, GitOps, or Terraform specified in the [upgrade cluster command]({{< relref "../cluster-upgrades/vsphere-and-cloudstack-upgrades/#upgrade-cluster-command" >}}) to apply those changes.
```

Comment:
```suggestion
Once you have made configuration updates, you can use `eksctl anywhere`, `kubectl`, GitOps, or Terraform specified in the [upgrade cluster command]({{< relref "../cluster-upgrades/vsphere-and-cloudstack-upgrades/#upgrade-cluster-command" >}}) to apply those changes.
```

nit: maybe specify `eksctl anywhere` command?

Reply:
i feel its ok. we have multiple places using `eksctl` solely and i personally think `eksctl` here is a different command line binary in comparison to `kubectl`. besides we have the example command in the upgrade command section showing the exact `eksctl` command we run.